aries-site/src/pages/templates/forms/managing-child-objects.mdx

```diff
@@ -0,0 +1,74 @@
+import { Example } from '../../../layouts';
+
+## When to use
+
+The managing child objects pattern may be used when creating, updating, or removing 
+child objects is part of a larger flow where a parent resource is being created or 
+configured. In these circumstances, the additional configuration inputs can crowd screens 
+and lead to quite lengthy forms.
+
+This pattern improves productivity by focusing a user's attention solely on the current object 
+of interest, and uses collapsible objects to show or hide detail as needed. The 
+intent is that collapsible objects reduce noise, clutter, and the resulting cognitive overload 
```

Comment:
```suggestion
intent is that collapsible objects reduce noise, clutter, and cognitive overload 
```
nit, but felt like could be simplified to this

Reply:
Like. 👍

Reply:
done